src/main/java/com/kruize/metrics/PodMetrics.java

```diff
@@ -111,8 +111,8 @@ public double getCurrentCpuLimit()
         return currentRecommendations.getCpuLimit();
     }
 
-    public double getCurrentRssLimit()
-    {
-        return currentRecommendations.getRssLimit();
-    }
+    public double getCurrentRssLimit() { return currentRecommendations.getRssLimit(); }
+
```

Comment:
Can you please delete the additional lines here.

Reply:
Done, Deleted as requested. Thanks!!